rslib/src/scheduler/states/relearning.rs

```diff
@@ -52,6 +52,14 @@ impl RelearnState {
                 },
             }
             .into()
+        } else if let Some(states) = &ctx.fsrs_next_states {
```

Comment:
Does this work if there are no memory states at that point of time (e.g. the deck was changed)?

Reply:
The state is recalculated prior to that point.